phoenix-core/src/main/java/org/apache/phoenix/jdbc/PhoenixDatabaseMetaData.java

```diff
@@ -219,6 +219,12 @@
     public static final byte[] TASK_TYPE_BYTES = Bytes.toBytes(TASK_TYPE);
     public static final String TASK_TS = "TASK_TS";
     public static final byte[] TASK_TS_BYTES = Bytes.toBytes(TASK_TS);
+    public static final String TASK_STATUS = "TASK_STATUS";
+    public static final String TASK_END_TS = "TASK_END_TS";
+    public static final String PRIORITY = "PRIORITY";
+    public static final String DATA = "DATA";
```

Comment:
Maybe make the constant named "TASK_DATA_COLUMN" or something along those lines to indicate what it's for?

Reply:
(and same with PRIORITY)

Reply:
Will rename as TASK_DATA. The others don't have _COLUMN at the end